connect/transforms/src/test/java/org/apache/kafka/connect/transforms/InsertFieldTest.java

```diff
@@ -16,7 +16,21 @@
  */
 package org.apache.kafka.connect.transforms;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Reordering the imports was not a necessary change. The previous ordering was consistent with the other related tests, so I would prefer this is reverted.

Reply:
I returned ordering back.